stormpath/stormpath-sdk-java
# Coverage clover tck

This PR causes the `spring-security-webmvc` and `spring-security-spring-boot-webmvc-bare-bones` examples to be run during IT time and get hit by TCK tests.

How to run these TCK tests:

STORMPATH_API_KEY_ID=XXXX STORMPATH_API_KEY_SECRET=YYYY mvn --fail-never -DskipITs=false -Pclover.all clean verify

* XXXX is the apiKeyID of a tenant with no additional apps than the 2 that are always present by default
* YYYY is the apiKeySecret of a tenant with no additional apps than the 2 that are always present by default

In order to run this you need to have `clover.license` under `~/.stormpath/clover/`.

What to check: Any of the Stormpath Jar files inside the `lib` directory of `stormpath-sdk-examples-spring-security-webmvc-1.0.RC9-SNAPSHOT.war` and `stormpath-sdk-examples-spring-security-spring-boot-webmvc-bare-bones-1.0.RC9-SNAPSHOT.jar` must have instrumented code.

BTW, TCK tests will fail as we are not spec complaint yet. We temporarily added `--fail-never` to TCK when it is run

fixes #501
